filament/backend/include/private/backend/HandleAllocator.h

```diff
@@ -36,6 +36,7 @@ namespace filament::backend {
 /*
  * A utility class to efficiently allocate and manage Handle<>
  */
+template <int P0, int P1, int P2>
```

Comment:
use `size_t` instead of `int` (or at least `unsigned`)

Reply:
Done.